### PR TITLE
Upgrade to Decidim v0.28

### PR DESCRIPTION
**Pull request by laurajaime**
## Release notes
https://github.com/decidim/decidim/releases/tag/v0.28.0
https://github.com/decidim/decidim/releases/tag/v0.28.1

Screenshot from 2024-06-20 13-32-33

Screenshot from 2024-06-20 13-32-21

### CODE REVIEW

File: .rubocop_ruby.yml [hunk 1]
```diff
@@ -70,11 +70,6 @@ AllCops:
   # Otherwise we fallback to the oldest officially supported Ruby version (2.0).
   TargetRubyVersion: 2.7
```
Comment (tramuntanal) [inline .rubocop_ruby.yml]: This is incorrect

File: config/application.example.yml [hunk 2]
```diff
@@ -36,9 +35,6 @@ test:
   <<: *default
   DB_TEST_DATABASE: "codit_demo_app-test"
 
-test:
-  <<: *default
```
Comment (tramuntanal) [inline config/application.example.yml:40]: why remove?

**Review by tramuntanal — COMMENTED** [review]
I've only seen just one thing that I think is incorrect to remove. Let me know

File: config/application.example.yml (hunk 2 above)
Comment (laurajaime) [reply config/application.example.yml:40]: I dont know why...because in local I have it..

Comment (laurajaime) [reply config/application.example.yml:40]: ahhhh, I see..it's duplicated...

image

**Review by tramuntanal — APPROVED** [review]
Good, LGTM now!

**Pull request merged**